genophase.wdl

```diff
@@ -81,6 +84,45 @@ workflow genophase {
     }
 }
 
+task concatenate_bams {
+    input {
+        Array[File] bams
+        Int disk_size_gb = 500
```

Comment:
I think this should be increased significantly. At least to 6000 to match `hic.merge`

Reply:
Also could you perhaps make disk size an input? I anticipate having to change it with respect to how many bams are inputted.

Reply:
I'll increase the default and make the disk size an optional workflow level input.